### ISSUE TITLE
Negative assertion messages don't work in jasmine

### ISSUE THREAD

**Issue by atomanyih** [blainekasten/enzyme-matchers#83]
```js
  it('renders the name', () => {
    expect(wrapper).not.toIncludeText('lars van der hoover');
  });
```

Failure message: `Expected <SuperCoolComponent> to contain "lars van der hoover" but it did not.`

Looks like https://github.com/blainekasten/enzyme-matchers/commit/5d4b5505c0e8ce0532a50aab2a7d73cfde07ea87 removed negated messages for jasmine

**Comment by blainekasten** [comment]
@atomanyih yikes! That's not good.. I did change things around how those things work and I primarily focused my effort around jest.. Sorry about that. 

Are you interested in trying to fix this?

**Comment by blainekasten** [comment]
@atomanyih this should be fixed now in 3.7.0! Give it a try and let me know
